celery/app/base.py

```diff
@@ -1249,14 +1252,32 @@ def amqp(self):
         """AMQP related functionality: :class:`~@amqp`."""
         return instantiate(self.amqp_cls, app=self)
 
+    def inform_green(self, is_green):
```

Comment:
Not a show-blocker, but I'd recommend adding a small unit test according to the code coverage warning